src/react/containers/EditorContainer.js

```diff
@@ -87,7 +87,7 @@ class EditorContainer extends Component {
   }
 
   publish() {
-    const { updateEntry, entry, entryEditClose, createEntry, isEditing } = this.props;
```

Comment:
Why are we removing entryEditClose?

Reply:
This commit explains it: https://github.com/Automattic/liveblog/pull/527/commits/0ef80388b0b6bc9e4faa9771c1562ded6ae8edcd

> This function was used to close the editor immediately after clicking 'Publish Update'. When a user on a slower network/server clicked this, the editor disappeared instantly, however it would take several seconds for the entries text to actually update - there was no visual feedback that their action was processing.
> 
> By not using this function the editor does not instantly get removed. Not being removed right away allows the editor component to check if the `store > user > entries` entry has a true isPublishing state, adding necessary indicators of progress.
> 
> The editor is already set to close by changing the isEditing state to false, on the `UPDATE_ENTRY_SUCCESS` Redux action in the user.js reducer.

Reply:
Ah yes @sboisvert, I see the problem now, I had that in the wrong commit. My mistake.